src/pages/Checkout.tsx

```diff
@@ -1,15 +1,24 @@
 import { PaymentSuccessCallback } from '../components/PaymentCard';
```

Comment:
The `Checkout` includes quit a lot of code. Although not necessary, you could consider splitting it up the code into different containers.